README.md

```diff
@@ -1,9 +1,9 @@
 # Elm loader [![Version](https://img.shields.io/npm/v/elm-webpack-loader.svg)](https://www.npmjs.com/package/elm-webpack-loader) [![Travis build Status](https://travis-ci.org/elm-community/elm-webpack-loader.svg?branch=master)](http://travis-ci.org/elm-community/elm-webpack-loader) [![AppVeyor build status](https://ci.appveyor.com/api/projects/status/7a5ws36eenwpdvgc/branch/master?svg=true)](https://ci.appveyor.com/project/elm-community/elm-webpack-loader/branch/master)
 
-[Webpack](http://webpack.github.io/docs/) loader for the [Elm](http://elm-lang.org/) programming language.
+[Webpack](https://webpack.js.org/) loader for the [Elm](http://elm-lang.org/) programming language.
 
 It is aware of Elm dependencies and tracks them. This means that in `--watch`
-mode, if you `require` an Elm module from a Webpack entry point, not only will
+mode, if you `require` an Elm module from a webpack entry point, not only will
```

Comment:
Recapitalised. Although for reference https://twitter.com/wSokra/status/855800490713649152